### PR TITLE
Fix author list; make changes due to P0009R11 changes

### PR DESCRIPTION
**Pull request by mhoemmen**
1. Fix Bryce's name and e-mail address, per his request today.
2. Revise proposal due to P0009 changes (esp. `ptrdiff_t` -> `size_t`).

### CODE REVIEW

**Comment by mhoemmen** [comment]
This will fix https://github.com/kokkos/stdBLAS/issues/73.

**Review by dalg24 — APPROVED** [review]

**Review by crtrott — APPROVED** [review]

File: D1673/blas_interface.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 ## Contributors
 
 * Chip Freitag (chip.freitag@amd.com) (AMD)
-* Bryce Lelbach (blelbach@nvidia.com) (NVIDIA)
+* Bryce Adelstein Lelbach (brycelelbach@gmail.com) (NVIDIA)
 * Srinath Vadlamani (Srinath.Vadlamani@arm.com) (ARM)
 * Rene Vanoostrum (Rene.Vanoostrum@amd.com) (AMD)
```
Comment (dsunder) [inline D1673/blas_interface.md:24]: Could you update my contact info to dansunderland@gmail.com?

**Review by dsunder — COMMENTED** [review]
The current version of P0009 in the mailings is R12, should we be rebasing to that instead?  I don't think it invalidates any of the changes you have made.

Also the next version of P0009 will be renaming basic_mdspan to mdspan and removing the mdspan template alias.  Assuming that both papers are in the next mailing, would it make sense to apply that change here as well?

File: D1673/blas_interface.md (hunk 1 above)
Comment (mhoemmen) [reply D1673/blas_interface.md:24]: Hi @dsunder ! : - )  I'll be happy to update your contact info.  I'll need to make another pass in any case to incorporate the P2299 changes (esp. `basic_mdspan` -> `mdspan`), so I can update the P0009 version then.

Comment (mhoemmen) [reply D1673/blas_interface.md:24]: @dsunder PR https://github.com/ORNL/cpp-proposals-pub/pull/144 updates your contact info -- please feel free to review if you like -- thanks! : - )

**Pull request merged**